File: wear/src/main/java/com/pdyjak/powerampwear/music_browser/folders/FolderViewHolder.java
package com.pdyjak.powerampwear.music_browser.folders;

import android.support.annotation.NonNull;
import android.view.View;

import com.pdyjak.powerampwear.music_browser.SimpleTwoLineItemViewHolderBase;

class FolderViewHolder extends SimpleTwoLineItemViewHolderBase {
    FolderViewHolder(View itemView) {
        super(itemView);
    }

    @Override
    public void bindWith(@NonNull Object object) {
        super.bindWith(object);
        FolderItem item = (FolderItem) object;
        mFirstLine.setText(item.name);
        mSecondLine.setText(item.parentName);
    }
}

File: wear/src/main/java/com/pdyjak/powerampwear/music_browser/folders/FolderViewHolder.kt
package com.pdyjak.powerampwear.music_browser.folders

import android.view.View

import com.pdyjak.powerampwear.music_browser.SimpleTwoLineItemViewHolderBase

internal class FolderViewHolder(itemView: View) : SimpleTwoLineItemViewHolderBase(itemView) {

    override fun bindWith(obj: Any?) {
        super.bindWith(obj)
        val item = obj as FolderItem
        mFirstLine.text = item.name
        mSecondLine.text = item.parentName
    }
}
